PyGame_Project/MVC/View_CLI/View.py

```diff
@@ -51,29 +50,20 @@ def print_current_puzzle():
         Score: {puzzle.score} / {puzzle.total_points} 
         Words Guessed: {prettyGuesses}
         
+        
                 ,---.
                /     \\
-          ,---<   {fir}   >---.
-         /     \     /     \\
-         "  {sec}  ">---<"  {thi}  "
-         \     /     \     /
-          >---<   {req}   >---<
-         /     \     /     \\
-         "  {fou}  ">---<"  {fif}  "
+          ,---<   {fir}   >---.                              
+         /     \     /     \\                         Puzzle Commands
+         "  {sec}  ">---<"  {thi}  "              --------------------------------------
+         \     /     \     /               /shuffle    /highscores    /help
+          >---<   {req}   >---<                /hints      /sharekey      /commands
+         /     \     /     \\               /showall    /shareimage    /mainmenu
+         "  {fou}  ">---<"  {fif}  "               /savegame   /giveup        /exit
          \     /     \     /
           `---<   {six}   >---'
                \     /
                 "---"
-
```

Comment:
I really like this feature, it's also different from what everyone else did. Can we make it three columns instead of two so it not "obnoxiously large"? We could show the few important ones like give up, shuffle and hits then make a commands option that lists all the options on the scree? (Could print out the command array you use for tab completion)